### DISCUSSION TITLE
Ethernet on, wifi off

### DISCUSSION THREAD

**Discussion by hsweet** [jamulussoftware/jamulus#2067] [Documentation]
A short warning to use only Ethernet or to turn off wifi in the instructions might  be useful.  

It seems that Jamulus can get confused if your computer has Ethernet AND wifi enabled.  It seems to cause 2 instances of a participant that come and go at random times, and messes up that user's sound temporarily.  I guess the local router switches interfaces from one to the other making one instance disconnect (but hang until timeout) and start a 2nd.

**Comment by dcorson-ticino-com** [comment]
What OS was the computer getting mixed up using?
I always have wifi and Ethernet both enabled using Win10 and have never had a problem.
When connected to Ethernet it simply doesn't use the wifi.

**Comment by hsweet** [comment]
I'm using Ubuntu 20.04.  But it happened to someone on a mac last week.  I can check to see if she had both interfaces running.  Jamulus 3.8 on the client and server for me.

**Comment by Andrew999999** [comment]
I use Jamulus on a PC that doesn't have wifi and have never experienced this issue - but I've seen band members who use both MacOS and Android occasionally appear more than once in the way you describe.  And they have both ethernet and wifi enabled.  We'll do some testing to see if your solution works for us.  Thanks for highlighting it.

**Reply by gene96817** [reply]
If you have both ethernet and wifi interfaces active on the client machine, it is important to configure the internal router to always send the Jamulus messages to the ethernet interface. I know that both MacOS and Win have router and firewall configurations. I don't have the patience to do that kind of work on my notebook. Theoretically, the OS router can choose to send some packets through ethernet or wifi as part of managing the buffers (I have watched that happen, just idle curiosity). The behavior could also change from release to release or depending on the set of processes running. Do you really want to be learning all those details? Better to just turn off the wifi interface when using Jamulus.

**Comment by hsweet** [comment]
Interesting, but I don't want to know that much. Or have to configure a
firewall to not allow packets on the Jamulus ports on the wi-fi address.

A line in the instructions to "Use Ethernet *and* turn off wi-fi"  would be
sufficient.

**Reply by gilgongo** [reply]
> A line in the instructions to "Use Ethernet *and* turn off wi-fi"  would be
sufficient.

I've added that in the next release of the website.
